src/app/components/character-count/index.html

```diff
@@ -0,0 +1,284 @@
+        </sky-demo-page-component-states>
+      </sky-demo-page-content>
+
+      <sky-demo-page-content
+        sectionHeading="Content"
+      >
+        <div
+          class="sky-text-block"
+        >
+          <p>
+            For the validation message when the character limit is exceeded, use the following format:
+          </p>
+          <p>
+            <b>Limit &lt;input field label&gt; to &lt;n&gt; characters.</b>
+          </p>
+        </div>
+      </sky-demo-page-content>
+
+      <sky-demo-page-content
+        sectionHeading="Layout"
+      >
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>
+              The character count is always located above the right side of the input field. If the field label is long, it wraps to the next line without displacing the character count.
+            </p>
+          </sky-docs-visual-example-text>
+
+          <sky-docs-visual-example-image>
+            <div>
+              <img
+                src="~/assets/img/guidelines/character-count/character-count-layout-1.png"
+              />
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+      </sky-demo-page-content>
+
+      <sky-demo-page-content
+        sectionHeading="Related information"
+      >
+        <h3>
+          Components
+        </h3>
+        <ul>
+          <li><a routerLink="/components/form">Form</a></li>
+          <li><a routerLink="/components/status-indicator">Status indicator</a></li>
+        </ul>
+
+        <h3>Guidelines</h3>
+        <ul>
+          <li><a routerLink="/design/guidelines/form-design">Form design</a></li>
+        </ul>
+      </sky-demo-page-content>
+    </sky-tab>
+
+    <sky-tab
+      tabHeading="Development"
+    >
+
+      <sky-demo-page-module-info
+        skyuxModule="SkyCharacterCounterModule"
+        packageName="@skyux/forms"
+        githubRepo="skyux-forms"
+      >
+      </sky-demo-page-module-info>
+
+      <sky-demo-page-properties
+        sectionHeading="Character count input directive properties"
+      >
+        <sky-demo-page-property
+          propertyName="skyCharacterCounter"
+          isOptional="false"
+        >
+          Creates an input that validates the number of characters in a value. Place this attribute on an <sky-code>input</sky-code> element. If users enter more characters than the provided limit, the input is invalid and an error indicator is displayed by the character count.
+        </sky-demo-page-property>
+        <sky-demo-page-property
+          propertyName="skyCharacterCounterIndicator"
+          isOptional="true"
+        >
+          The character count indicator component. This is the component that displays the count, limit, and over the limit indicator.
+        </sky-demo-page-property>
+        <sky-demo-page-property
+          propertyName="skyCharacterCounterLimit"
+          defaultValue="0"
```

Comment:
Does this limit mean that if users don't specify a limit, then the field errors as soon as users enter anything at all? If so, that seems like weird default to me.

Reply:
Yeah, it was the default Jack had set but it definitely seems odd. What do we think the default should be, or should it just be a required field?

Reply:
IMO should be required if using the counter, otherwise there's not much point to using it.

Reply:
Agreed, a count limit must be provided to use this. No default will make enough sense to risk encouraging laziness.

Reply:
So @Blackbaud-AdamFunderburk and @Blackbaud-ToddRoberts, am I correct in assuming that you want us to remove the default on this property and make it required? This would be considered a breaking change, but the component isn't officially released yet so I don't think that matters. Besides, it's highly unlikely that even if anyone was already using it that they would be using the component with the default value of 0. @Blackbaud-SteveBrush, do you have any opinion on this?

Reply:
I updated this to hide the default and marked the property as required after discussing this with the team.